worldwind/src/commonMain/kotlin/earth/worldwind/WorldWind.kt

```diff
@@ -494,6 +496,8 @@ open class WorldWind @JvmOverloads constructor(
         // Reset the render context's state in preparation for the next frame.
         rc.reset()
 
+        ++frameIndex
```

Comment:
Can we inline this increment with assigment?

Reply:
done